guide/README.md

```diff
@@ -455,6 +455,56 @@ used to bump the `Mi 4` note up by one octave and then to bump the
 | | | For example: Transposing C-D-E-F by 4 (fifth) will give us G-A-B-C
 | | | To transpose an octave: shift by the mode length (7 in major scales) up or down. |
 
+#### <a name="Set Key"></a>3.2.5 Set Key
+
+![alt tag](./setkey1.svg "scalar transposition")
+
+
+The Set key block is used to set the key and mode, eg C Major. 
+It allows you to change both the mode and key of how solfege is mapped to the notes.
+
+
+| Standard Notation for Set Key |
+| --- |
+| ![alt tag](./setkey2.png "Standard Notation Set Key") |
+
+[RUN LIVE](https://musicblocks.sugarlabs.org/index.html?id=1662103714150464&run=True)
+
+In set key , mode determines the pattern of notes and key determines where the pattern starts. In the following table it is shown how notes and modes pattern changes with changing value of set key value.
```

Comment:
When using the *Set key* block, the mode argument is used to define the pattern of half steps and the key argument is used to define the starting position of the pattern. For example, when mode = "major" and key = "C", the pattern of half steps is 2 2 1 2 2 2 1 and the first note in the scale from which the pattern is applied is "C".

Examples are shown in the following table.